src/cli/commands/graph.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import {Command, flags} from "@oclif/command";
+import mkdirp from "mkdirp";
+import os from "os";
+import path from "path";
+
+const execDependencyGraph = require("../../tools/execDependencyGraph").default;
+
+export default class GraphCommand extends Command {
+  static description = "create the contribution graph for a repository";
+
+  static args = [
+    {
+      name: "repo_owner",
+      required: true,
+      description: "owner of the GitHub repository for which to fetch data",
+    },
+    {
+      name: "repo_name",
+      required: true,
+      description: "name of the GitHub repository for which to fetch data",
+    },
+  ];
+
+  static flags = {
+    "output-directory": flags.string({
+      short: "o",
+      description: "directory into which to store graphs",
```

Comment:
In our previous example,we computed an appropriate temporary output dir and then set it as the default here, which would make the command usage more helpful. is there a reason not to do that here?

Reply:
Also: `"env": SOURCECRED_OUTPUT_DIRECTORY`? :smiley:

Reply:
> In our previous example,we computed an appropriate temporary output dir and then set it as the default here, which would make the command usage more helpful. is there a reason not to do that here?
 
On line 49 (`const outputDirectory = …`) below, we do do that. We do not set an explicit default value in the flags configuration because the default value depends on the repository name.

Reply:
> Also: `"env": SOURCECRED_OUTPUT_DIRECTORY`? :smiley: 

Sure, can do.

Reply:
> > In our previous example,we computed an appropriate temporary output dir and then set it as the default here, which would make the command usage more helpful. is there a reason not to do that here?
>
> On line 49 (`const outputDirectory = …`) below, we do do that. We do not set an explicit default value in the flags configuration because the default value depends on the repository name.

I'll change this to have the following semantics. The output directory defaults to `path.join(os.tmpdir(), "sourcecred")`, and graphs are always put in `path.join(outputDirectory, repoOwner, repoName)`, regardless of whether `outputDirectory` is manually set or the default value. This seems like better behavior.

Reply:
:+1: